lib/mix/tasks/compile.make.ex

```diff
@@ -186,6 +186,14 @@ defmodule Mix.Tasks.Compile.ElixirMake do
     error_msg = Keyword.get(config, :make_error_message, :default) |> os_specific_error_msg()
     custom_args = Keyword.get(config, :make_args, [])
 
+    if String.contains?(cwd, " ") do
+      IO.warn(
+        "the absolute path to the makefile for this project contains spaces. Make might " <>
```

Comment:
```suggestion
        "the absolute path to the Makefile for this project contains spaces. \"make\" might " <>
```

Reply:
We support executables that are not `make` (like `nmake`) and makefiles that are not called `Makefile`, that's why I went with this naming 😄